ods_ci/libs/DataSciencePipelinesAPI.py

```diff
@@ -0,0 +1,242 @@
+import subprocess
+from json import JSONDecodeError
+import requests
+import os
+import json
+import time
+from robotlibcore import keyword
+import base64
+
+
+class DataSciencePipelinesAPI:
+
+    def __init__(self):
+        self.auth_url = self.retrieve_auth_url()
+        self.route = ''
+        self.sa_token = None
+
+    @keyword
+    def create_operator(self, file):
```

Comment:
As a suggestion for a future PR, this method could be implemented as regular Robot Framework keyword using the robotframework-openshift library.